packages/ui/src/components/editor/properties/collider/index.tsx

```diff
@@ -48,13 +51,18 @@ export const ColliderComponentEditor: EditorComponentType = (props) => {
   const { t } = useTranslation()
   const colliderComponent = useComponent(props.entity, ColliderComponent)
 
+  const childMeshEntities = useChildrenWithComponents(props.entity, [MeshComponent])
+
   return (
     <NodeEditor
       {...props}
       name={t('editor:properties.collider.name')}
       description={t('editor:properties.collider.description')}
       Icon={ColliderComponentEditor.iconComponent}
     >
+      {childMeshEntities.length == 0 && ( //<ErrorPopup className="ml-5 text-red-400" visibleDuration={10000}  >{ t('editor:properties.collider.warn-missing-mesh') }  </ErrorPopup>
```

Comment:
i think this limitation will break expectations. we dont need a mesh (that is then set to invisible) just to add a collider - we should be able to just add a collider to an entity.

Reply:
this is a good point, maybe we only show this in mesh mode. This discussion does however make me feel like we should include an option in the node editor to attempt to base the collider off of a mesh or not